@here/harp-map-controls/lib/MapAnimations.ts

```diff
@@ -452,7 +452,9 @@ export class CameraPanAnimation extends CameraAnimation {
             })
             .onUpdate(({ latitude, longitude, altitude }) => {
                 this.mapView.geoCenter = new GeoCoordinates(latitude, longitude, altitude);
-                this.mapView.camera.position.z = altitude;
+                if (altitude) {
```

Comment:
this will not set the altitude if its 0, this changes the former behaviour... why is this change needed?